src/rancilio-pid.cpp

```diff
@@ -1423,22 +1430,27 @@ void machinestatevoid() {
 
         case kBrew:
             brewdetection();
+            // Ausgabe waehrend des Bezugs von Bruehzeit, Temp und heatrateaverage
```

Comment:
It seems you've reverted some code back to an old state (incl. german comment etc.)